README.md

```diff
@@ -59,6 +59,8 @@ and it shares the same defaults and behavior as stashcp.
 Building
 --------
 
+If the `parameters.yaml` file was altered, regenerate `parameters.go` by running `go generate` in the `./config` directory
```

Comment:
Is this comment needed?  I believe `goreleaser` (and just hitting `make`) automatically runs `go generate`, no?

Reply:
Removed from the code